sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/NamedVIPEvaluationStage.java

```diff
@@ -32,12 +32,15 @@ public NamedVIPEvaluationStage(
             String protocol,
             DiscoveryInfo.Visibility visibility,
             String vipName,
-            Integer vipPort) {
-        super(resource, taskName, portName, port, customEnvKey);
+            Integer vipPort,
+            boolean useHostPorts,
+            boolean onOverlay) {
+        super(resource, taskName, portName, port, customEnvKey, useHostPorts);
         this.protocol = protocol;
         this.visibility = visibility;
         this.vipName = vipName;
         this.vipPort = vipPort;
+        this.onOverlay = onOverlay;
```

Comment:
@gabrielhartmann after your change goes in, this field won't need to be duplicated here since it will accessible from the `NamedVIPSpec` that gets passed in to this evaluation stage, correct?